examples/src/bin/rng.rs

```diff
@@ -0,0 +1,39 @@
+//! RNG test
```

Comment:
I'd probably sack this example, it's probably too trivial to be more useful than the docs.